addons/project/views/project_views.xml

```diff
@@ -1181,13 +1177,9 @@
                                     <field name="company_id" groups="base.group_multi_company" optional="hide"/>
                                     <field name="company_id" invisible="1"/>
                                     <field name="activity_ids" string="Next Activity" widget="list_activity" optional="hide"/>
-                                    <field name="date_deadline" attrs="{'invisible': [('is_closed', '=', True)]}" optional="show"/>
+                                    <field name="date_deadline" attrs="{'invisible': [('is_closed', '=', True)]}" optional="hide"/>
                                     <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color'}" optional="hide"/>
-                                    <field name="rating_last_text" string="Rating" decoration-danger="rating_last_text == 'ko'"
```

Comment:
It is better to add `invisible="1"` on `rating_last_text` instead of removing it. 🙂

Reply:
Done

Reply:
Nope :)